Tests/runTestSimulation.m

```diff
@@ -110,23 +105,22 @@
 S.misc.main_path = pathRepo;
 
 
-if ~isempty(PathCpp2Dll_Exe)
-    S.Cpp2Dll.PathCpp2Dll_Exe = InstallOsim2Dll_Exe(PathCpp2Dll_Exe);
-end
 if ~isempty(CasADi_path)
     S.solver.CasADi_path = CasADi_path;
 else
     S.solver = rmfield(S.solver,'CasADi_path');
 end
-if ~isempty(Cpp2Dll_compiler)
-    S.Cpp2Dll.compiler = Cpp2Dll_compiler;
+if ~isempty(OpenSimAD_compiler)
+    S.Cpp2Dll.compiler = OpenSimAD_compiler;
 else
     S.Cpp2Dll = rmfield(S.Cpp2Dll,'compiler');
 end
 if max_iter > 0
     S.solver.max_iter = max_iter;
 end
 
+S.OpenSimADOptions.verbose_mode = 0; % 0 for no outputs from cmake
```

Comment:
Might be good to have verbose mode on for tests. Or make it function input?

Reply:
added it as input